examples/e-commerce/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "lib": [
+      "es6",
+      "dom",
+      "dom.iterable"
+    ],
+    "downlevelIteration": true,
+    "esModuleInterop": true
```

Comment:
I don't know why we didn't need this in the root config, but I need it for the sandbox to work